mantle/cmd/kola/resources/iscsi_butane_setup.yaml

```diff
@@ -97,8 +97,14 @@ storage:
           # Start by default on boot
           WantedBy=multi-user.target
           [Service]
+          # We want ExecStartPost to be run after the podman container have actually started
+          # The default type runs post commande once the process have started, rather than waiting for the exit code
+          Type=oneshot
           # fix permissions on the serial device before passing it as a volume
           ExecStartPre=chmod 777 /dev/virtio-ports/testisocompletion
+          # Pipe the logs to a virtio port so kola saves them
+          StandardOutput=file:/dev/virtio-ports/nestedvmlogs
+          ExecStartPost=-podman logs -f iscsiboot 2> /dev/virtio-ports/nestedvmlogs
```

Comment:
I wonder if setting `LogDriver=passthrough` in the `[Container]` section above would give us what we want here instead of having to run a separate command.

Reply:
Also maybe we should add a line for StandardError with a comment

```
# Make StandardError inherit from StandardOutput (will use same fd for output)
StandardError=inherit
StandardOutput=file:/dev/virtio-ports/nestedvmlogs
```

Reply:
I was not aware of this passthrough log driver. It looks like it could do the trick, let me try that

Reply:
somewhat related : that seems to be the default behavior in more recent podman/quadlet + systemd : i can see logs from the container in journalctl, but i don't see the `log-driver` command in the generated unit

Reply:
yep that works ! Thanks for the tip